### PR TITLE
Feature/ctech 2907 use poetry to define dependencies and publish project

### PR DESCRIPTION
**Pull request by adamMQM**
# Pull Request Checklist

- [x] Read the [contributing guidelines](https://github.com/finbourne/lusid-python-tools/blob/master/docs/CONTRIBUTING.md)
- [x] Tests pass

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+[tool.poetry]
+name = "lusidtools"
+version = "2.0.0a0"
```
Comment (rizwansaeed) [inline pyproject.toml:3]: does this need to go to just "2.0.0"?

**Comment by rizwansaeed** [comment]
did the black formatting get wired in?

File: pyproject.toml (hunk 1 above)
Comment (adamMQM) [reply pyproject.toml:3]: This is the prerelease tag - the pipeline will patch it to 2.0.0 automatically before deployment.

This way the version in source control matches the deployed version

**Comment by adamMQM** [comment]
> did the black formatting get wired in?

no - will do as a separate piece of work and will black format entire project, as a lot of files are now incorrectly formatted.

**Pull request closed without merging**